db/migrate/20240517121235_require_tax_category_on_tax_rate.rb

```diff
@@ -0,0 +1,5 @@
+class RequireTaxCategoryOnTaxRate < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :spree_tax_rates, :tax_category_id, false
```

Comment:
I double-checked the assumption on all prod servers:
```
sql='select count(*) from spree_tax_rates where tax_category_id is null;'
ansible all_prod -u openfoodnetwork -a "psql -h localhost openfoodnetwork ofn_user -c '$sql'"
```

All clear! :heavy_check_mark: